### PR TITLE
lorem: add page

### PR DESCRIPTION
**Pull request by AnimiVulpis**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/lorem.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# lorem
+
+> Create more or less random lorem ipsum text.
+
+- Print the number of words you want:
```
Comment (sbrl) [inline pages/common/lorem.md]: Generally, we try to avoid using `you` in tldr-pages. Perhaps we can rephrase this?
```

Comment (pxgamer) [reply pages/common/lorem.md]: Perhaps:
```suggestion
- Print the specified number of words:
```

File: pages/common/lorem.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Print 10 lines of Goethe's Faust:
+
+`lorem -l {{10}} --faust`
+
+- Print 5 sentences of Poe's Raven:
+
+`lorem -s {{5}} --raven`
+
+- Print random words from Boccaccio's Decameron for 40 characters:
```
Comment (agnivade) [inline pages/common/lorem.md]: I would keep it consistent with the previous descriptions - 

"Print 40 random characters from Boccacio's Decameron".

File: pages/common/lorem.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`lorem --decamerone --randomize -c {{40}}`
```
Comment (agnivade) [inline pages/common/lorem.md]: Let us move `decamerone` to the end.

File: pages/common/lorem.md (hunk 2 above)
Comment (AnimiVulpis) [reply pages/common/lorem.md]: My idea was to show that the order options does not matter.
But the inconsistency might be more confusing than helping.

**Pull request merged**